src/hotspot/cpu/ppc/relocInfo_ppc.cpp

```diff
@@ -58,7 +58,9 @@ void Relocation::pd_set_data_value(address x, intptr_t o, bool verify_only) {
       assert(type() == relocInfo::oop_type || type() == relocInfo::metadata_type,
              "how to encode else?");
       narrowOop no = (type() == relocInfo::oop_type) ?
-          CompressedOops::encode((oop)x) : CompressedKlassPointers::encode((Klass*)x);
+          CompressedOops::encode((oop)x) :
+          // FIXME: Cheating! Treating narrowKlass as a narrowOop.
```

Comment:
I think we try to avoid adding new FIXMES and TODOS to the code. Maybe create a JBS entry instead?